### ISSUE TITLE
add 'spark_session' as alias for 'hive_context' (Spark 2.0.0)

### ISSUE THREAD

**Issue by kevinushey** [rstudio/sparkapi#12]
With Spark 2.0.0, the Spark session / SQL context / hive context have all been amalgamated into a single object. Right now, `hive_context()` returns that:

```
> hive_context(sc)
<jobj[8]>
  class org.apache.spark.sql.SparkSession
  org.apache.spark.sql.SparkSession@65b602ec
```

But we should be able to access this with a plain `spark_session(sc)` call.

**Comment by jjallaire** [comment]
Definitely! If we do this then we should also check for spark_version >= 2.0 and throw a nice error if it isn't a version that supports SparkSession
